connectors/rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/ReplicatorSourceConnector.java

```diff
@@ -208,22 +208,22 @@ public Class<? extends Task> taskClass() {
         return ReplicatorSourceTask.class;
     }
 
-    private Set<String> neededParamKeys = new HashSet<String>() {
+    private Map<String, Boolean> neededParamKeys = new HashMap<String, Boolean>() {
         {
-            add(ReplicatorConnectorConfig.SRC_CLOUD);
-            add(ReplicatorConnectorConfig.SRC_REGION);
-            add(ReplicatorConnectorConfig.SRC_CLUSTER);
-            add(ReplicatorConnectorConfig.SRC_ENDPOINT);
-            add(ReplicatorConnectorConfig.SRC_TOPICTAGS);
-            add(ReplicatorConnectorConfig.DEST_CLOUD);
-            add(ReplicatorConnectorConfig.DEST_REGION);
-            add(ReplicatorConnectorConfig.DEST_CLUSTER);
-            add(ReplicatorConnectorConfig.DEST_ENDPOINT);
-            add(ReplicatorConnectorConfig.DEST_TOPIC);
-            add(ReplicatorConnectorConfig.SRC_CLOUD);
-            add(ReplicatorConnectorConfig.SRC_ACL_ENABLE);
-            add(ReplicatorConnectorConfig.DEST_ACL_ENABLE);
-            add(ERRORS_TOLERANCE_CONFIG);
+            put(ReplicatorConnectorConfig.SRC_CLOUD, false);
```

Comment:
It is also possible to remove them directly, but it is easier for me to extend or customize them. It would be better to extract the required parameters into a unified public class rather than write them for each connector